### DISCUSSION TITLE
Increase HLS part number

### DISCUSSION THREAD

**Discussion by tschettervictor** [bluenviron/mediamtx#3143] [Questions]
### Question

Is there any way to increase the number of HLS parts the server will generate? I tried setting the segments to 20 and up, but it seems the HLS only generates 2 or 3 parts, which makes streaming for slower clients a bit laggy and the video and audio cuts out quite a bit.

**Comment by tschettervictor** [comment]
Follow up on this. It seems my real problem is this. 
When I open a stream via HLS, the playback speed is set to 1.5 in my browser, and I can't change it. Only when it press the (back 15 sec) button will the speed go to 1, and even then I can't change it. Is this expected behaviour? @aler9

**Reply by tschettervictor** [reply]
And it seems to only be happening on mobile devices. Windows computer looks like it plays fine.

**Reply by tschettervictor** [reply]
Just tested again. iPad with iOS 17 side by side with windows 11 laptop. The laptop had no issues, but the iPad played at 1.5x speed. Had to go back 10 seconds and it played at normal speed. But when going to the latest part (within 1-3 sec) it again goes to 1.5x speed.

Streaming with both rtmp/rtsp protocols from multiple different apps/services like OBS/Larix etc…

**Comment by aler9** [comment]
Hello, if the problem persists open a bug report and provide a sample file that allows to replicate the issue.

[a comment by an automated account is not shown]
